### PR TITLE
Readme tidyup

### PR DESCRIPTION
**Pull request by wilsondennison**
Readme tidyup

### CODE REVIEW

**Comment by McDonnellDean** [comment]
Checked.

File: README.md [hunk 1]
```diff
@@ -1,10 +1,23 @@
...
 
-Artistic License 2.0
+# Amazon Elastic Load Balancer Container
+Manages the creation, deployment, linking and unlinking of an Amazon Elastic Load Balancer for __nscale__. The
+aws-elb-container is used internally by __nscale__, see __nscale__'s [documentation] for information on how
+to deploy using Amazon Web Services as the target platform.
+
+
+## License
+Copyright (c) 2015 [nearForm] and other contributors
```
Comment (mcollina) [inline README.md]: The copyright year should be 2014-2015.

Comment (McDonnellDean) [inline README.md]: @wilsondennison see note on ami container

Comment (wilsondennison) [inline README.md]: No problem :) Will do

**Comment by wilsondennison** [comment]
@McDonnellDean  done :)

**Comment by mcollina** [comment]
Thanks for all your work, @wilsondennison!

**Comment by wilsondennison** [comment]
@mcollina  no problem. Enjoying helping out

**Pull request merged**